### PR TITLE
Update padlocal version

### PR DESCRIPTION
**Pull request by padlocal**
## Checklist

- [x] Commit Messages follow the [Conventional Commits](https://conventionalcommits.org/) pattern
  - A feature commit message is prefixed "feat:"
  - A bugfix commit message is prefixed "fix:"
- [x] Tests for the changes have been added
- [x] CI has been passed. (GitHub actions all turns green)
- [x] SLA has been signed

## Description

Update  wechayt-puppet-padlocal vesion from **0.4** to **0.4.1**, which fix PadLocal sign on related issues.

### CODE REVIEW

File: examples/ding-dong-bot.ts [hunk 1]
```diff
@@ -34,13 +34,13 @@ import { generate } from 'qrcode-terminal'
  */
 const bot = new Wechaty({
   name : 'ding-dong-bot',
-  
+
```
Comment (huan) [inline examples/ding-dong-bot.ts:37]: Please remove unrelated changes: keep the blank link between those comments to make them separate clearer.

**Review by huan — CHANGES_REQUESTED** [review]
Thanks for the updating!

Please remove the unrelated changes then this PR will be good to be merged!

File: examples/ding-dong-bot.ts (hunk 1 above)
Comment (padlocal) [reply examples/ding-dong-bot.ts:37]: Thanks Huan, I undestand. But following lint issue stopped me while pushing:

/Users/.../github/wechaty/examples/ding-dong-bot.ts
  37:1  error  Trailing spaces not allowed  no-trailing-spaces
  43:1  error  Trailing spaces not allowed  no-trailing-spaces

And there are actually no trail spaces on other empty lines in this file, so I updated code to fix the lint issue.
What's your suggestion for next step then?

**Review by huan — APPROVED** [review]
LGTM

File: examples/ding-dong-bot.ts (hunk 1 above)
Comment (huan) [reply examples/ding-dong-bot.ts:37]: That's OK, let's merge this update first.

Thank you very much!
